### ISSUE TITLE
Not possible to install / uninstall extension via keyboard

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#7966]
I like how the keydown starts going through the list and how enter opens the extension description.
I would exprect that an enter on already selected list item installs / uninstalls that extension.

The alternative is that you open extension description on each up / down and on enter install / uninstall. I think this might also work nicely

**Comment by isidorn** [comment]
Tab now moves focus out of search box, while the click anywhere in the viewlet still focuses the search box. Unforunetetly I did not find a differentiator in the `FocusEvent` so I had to add another listener for a press of a tab key.

**Comment by bpasero** [comment]
I find the current way very confusing if I compare it to the tree. This really needs some fixing in the list widget. I will open a separate issue.

**Comment by isidorn** [comment]
Agree - create me an issue and I will look into it next week. Thanks!

**Comment by bpasero** [comment]
#8621 and #8622

**Comment by stereokai** [comment]
Is there a particular reason there's no command for removing an extension?

**Comment by joaomoreno** [comment]
Not really, what is your use case?

**Comment by stereokai** [comment]
I want to be able to remove extension using only the keyboard. I don't think it deserves its own keybinding, but rather a command that's accessible via the command palette.

**Comment by joaomoreno** [comment]
You should be able to do that in the latest insiders:

- <kbd>Cmd Shift X</kbd>
- <kbd>Tab</kbd> <kbd>Tab</kbd>
- You should now be focused on the first extension's uninstall button.

Not a command sure, but definitely keyboard accessible.

We can think about commands yeah... if you'd make a PR for it, I'd gladly review it! It would need some sort of interaction with the quick open pick API in order to have the user pick which extension he'd like to uninstall.

**Comment by stereokai** [comment]
@joaomoreno what is the quick open pick API?

**Comment by joaomoreno** [comment]
[Here's a good example](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/parts/themes/electron-browser/themes.contribution.ts#L79)
